### ISSUE TITLE
Multi-cursor support

### ISSUE THREAD

**Issue by PEZ** [simonacca/tASTe#18]
As we are in the process of adding multi-cursor support to Calva Paredit (spearheaded by @riotrah) I can recommend taking on this early. We have it working for selections and movement and it is a major productivity boost.

**Comment by riotrah** [comment]
Yeah seriously - it's possibly going to be much harder to add multi cursor later vs building everything with it in mind!

**Comment by riotrah** [comment]
This giant old PR will have more examples of what multicursor might look like for each type of edit/movement etc. 

https://github.com/BetterThanTomorrow/calva/pull/1606
